frontend/src/locales/fr.json

```
@@ -34,10 +35,11 @@
   "contribution": {
     "activity": "Activité",
     "alert": {
+      "answerQuestion": "Please answer the question",
       "communityNoteList": "Vous trouverez ci-contre toutes les contributions versées et certifiées de tous les membres de cette communauté.",
       "confirm": " Approuvé",
+      "in_progress": "Es gibt eine Rückfrage der Moderatoren.",
```

```suggestion
      "in_progress": "There is a question from the moderators.",
```

May @Elweyn  could translate it?